src/SubscribeMe/Subscriber/MailjetSubscriber.php

```diff
@@ -52,14 +61,19 @@ public function subscribe(string $email, array $options, array $userConsents = [
 
         $uri = 'https://api.mailjet.com/v3/REST/contactslist/' . $this->getContactListId() . '/managecontact';
         try {
-            $res = $this->getClient()->request('POST', $uri, [
-                'http_errors' => true,
-                'auth' => [$this->getApiKey(), $this->getApiSecret()],
-                'headers' => [
-                    'Content-Type' => 'application/json',
-                ],
-                'body' => json_encode($body)
-            ]);
+            if (!is_string(json_encode($body))) {
```

Comment:
You can use `json_encode($body, JSON_THROW_ON_ERROR)` instead of testing if it's not a `string`.